test_files/lobster/lobsterouts/lobsterout.crash

```diff
@@ -0,0 +1,42 @@
+LOBSTER v3.2.0 (g++ 5.4.0)
```

Comment:
Not sure what this file is. Or the .keep file.

Reply:
will remove the .crash file - the .keep was there to create the folder in the first place. I need the empty folder for the test.

Reply:
Do you have a suggestion how I can add an empty folder to git without the .keep file? Thanks!

Reply:
This is fine. But why do you need an empty folder for the test?

Reply:
You are correct. I just need a folder without a "lobsterout" file to check that the validator is working. I will use one without lobsterout instead. Thanks.